esm/html.js

```diff
@@ -26,10 +26,8 @@ export const html = (query, ...args) => {
   return element;
 };
 
-html.extend = function (query, ...args) {
-  const clone = memoizeHTML(query);
-
-  return html.bind(this, clone, ...args);
+html.extend = (query, ...args) => {
+  return html.bind(this, memoizeHTML(query), ...args);
```

Comment:
`=>` messes up `this`, I’m not sure if that’s wise here.. 🤔

Reply:
Oh sorry, adjusting now. Was rushing through it, really think VSCode should add warnings for edge cases like this.